### PR TITLE
fix post-upgrade-hook imagePullSecretes

### LINKED ISSUE #289: post-upgrade-hook.yaml missing imagePullSecret
post-upgrade-hook.yaml missing the following lines, which result in imagePull failure when a private image registry is been used. 

      {{- if .Values.image.imagePullSecrets }}
      imagePullSecrets:
        {{ toYaml .Values.image.imagePullSecrets | indent 8 }}
      {{- end }}

### PR DESCRIPTION
**Pull request by bingwei-hong-partior**
Signed-off-by: bingwei-hong-partior <bingwei.hong@partior.com>

#### What this PR does / why we need it:
 This is to fix post-upgrade-hook.yaml missing imagePullSecrets tag when using private image registry that requires authentication.
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #289 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] DCO signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Review by gdsoumya — APPROVED** [review]
Thanks for the fix! LGTM

**Comment by gdsoumya** [comment]
Oh just noticed you forgot to bump the chart version and regenerate the docs. @bingwei-hong-partior can you bump the version and generate the docs

File: charts/litmus/templates/post-upgrade-hook.yaml [hunk 1]
```diff
@@ -23,6 +23,10 @@ spec:
         helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     spec:
       restartPolicy: {{ .Values.upgradeAgent.controlPlane.restartPolicy }}
+      {{- if .Values.image.imagePullSecrets }}
+      imagePullSecrets:
+        {{ toYaml .Values.image.imagePullSecrets | indent 8 }}
```
Comment (Jasstkn) [inline charts/litmus/templates/post-upgrade-hook.yaml:28]: It should be `nindent` here instead of indent.

**Review by Jasstkn — CHANGES_REQUESTED** [review]

**Comment by Jasstkn** [comment]
@bingwei-hong-partior hello! thanks for your contribution. One of the jobs is failed, to fix this you need to run `helm-docs`. This step is mentioned in our [contribution guide](https://github.com/litmuschaos/litmus-helm/blob/master/CONTRIBUTING.md#how-to-contribute). 

Let me know if you have questions!

File: charts/litmus/templates/post-upgrade-hook.yaml (hunk 1 above)
Comment (bingwei-hong-partior) [reply charts/litmus/templates/post-upgrade-hook.yaml]: OK, will change.

**Comment by bingwei-hong-partior** [comment]
Hi, I've done the chart version bump, docs update and use nindent . anything else needed?

**Comment by bingwei-hong-partior** [comment]
Sorry, first time contributing, appologise if I didn't get all process correctly

**Comment by Jasstkn** [comment]
> Hi, I've done the chart version bump, docs update and use nindent . anything else needed?

nope, it fails for another chart.

**Comment by Jasstkn** [comment]
> Sorry, first time contributing, appologise if I didn't get all process correctly

it's fine. let me check...

File: charts/litmus/README.md [hunk 2]
```diff
@@ -233,5 +233,3 @@ We separated service configuration from `portal.server.service` to `portal.serve
 | upgradeAgent.resources | object | `{}` |  |
 | upgradeAgent.tolerations | list | `[]` |  |
 
-----------------------------------------------
```
Comment (Jasstkn) [inline charts/litmus/README.md:236]: was the line delete automatically? it shouldn't

Comment (bingwei-hong-partior) [reply charts/litmus/README.md:236]: Hmm it was generated by helm-docs

Comment (Jasstkn) [reply charts/litmus/README.md:236]: I ran helm-docs from your branch and it reverts deletion of this line. I use 
```
helm-docs --version
helm-docs version 1.11.0
```

Comment (bingwei-hong-partior) [reply charts/litmus/README.md:236]: image
image
OK, regenerated using version 1.11.0

Comment (bingwei-hong-partior) [reply charts/litmus/README.md]: Hi @Jasstkn , is everything good? anything else I need to change?

Comment (gdsoumya) [reply charts/litmus/README.md]: Everything seems to be passing, all good.

**Review by Jasstkn — APPROVED** [review]
@bingwei-hong-partior thank again for your contribution! It looks good to me!

**Pull request merged**